### PR TITLE
Fix: Always resolve symlink in self.file_path and self.repo_path

### PR DESCRIPTION
**Pull request by randy3k**
I always encounter random errors when the working repo is a symlink. Always resolving symlink should fix the issue.

### CODE REVIEW

File: core/commands/log_graph.py [hunk 1]
```diff
@@ -39,8 +39,7 @@ def get_graph_args(self):
         savvy_settings = sublime.load_settings("GitSavvy.sublime-settings")
         args = savvy_settings.get("git_graph_args")
         if self._file_path:
-            file_path = self._file_path
-            file_path = os.path.realpath(file_path)[len(self.repo_path)+1:]
+            file_path = self.self._file_path(self._file_path)
```
Comment (stoivo) [inline core/commands/log_graph.py]: self.self?

Comment (randy3k) [inline core/commands/log_graph.py]: good catch

**Comment by stoivo** [comment]
LGTM ✨

**Comment by divmain** [comment]
LGTM.  Feel free to merge this @randy3k.  You may have to accept the repo invite but, once you do, you should have merge rights.

**Comment by randy3k** [comment]
For consistency with diff and inline diff,I have also refactored some code to use `git_savvy.file_path` to store current file name.

**Pull request merged**